### ISSUE TITLE
loadtest multi-scenario throw intermittent Fatal RPython error: AssertionError

### ISSUE THREAD

**Issue by rpappalax** [mozilla-services/ap-loadtester#55]
Seems more likely to happen when there is a larger number of instances spawned.  
This happened for me on OSX (haven't yet tried it on Linux)

Example:
~~~
$ aplt_testplan wss://autopush.stage.mozaws.net/ "aplt.scenarios:loadtest,20,1,0"
~~~

LOG
~~~
.net/wpush/v1/gAAAAABXkBG2J5xr9Zi_92cRyigGJQXT7OhKS_TD4f2Wc-LRoqhGBBgzzI5zU_0o0Z9pTERn_oiGU0FZrfoU-33Gt3DMXXRtknHeUQ8x06TqoRZcUzQ_nKD9XIGVLJRo707lX9lfC8D4'}
INFO:twisted:Running command:  Wait(time=2)
INFO:twisted:Running command:  Counter(name='notification.throughput.bytes', count=3111)
INFO:twisted:Running command:  Counter(name='notification.sent', count=1)
INFO:twisted:Running command:  Wait(time=2)
INFO:twisted:Running command:  Counter(name='notification.throughput.bytes', count=2922)
INFO:twisted:Running command:  Counter(name='notification.sent', count=1)
INFO:twisted:Running command:  Wait(time=2)
RPython traceback:
  File "rpython_jit_metainterp_7.c", line 49173, in MIFrame_run_one_step
  File "rpython_jit_metainterp_9.c", line 61354, in handler_residual_call_ir_i_1
  File "rpython_jit_metainterp_11.c", line 66643, in MIFrame_do_residual_call
  File "rpython_jit_metainterp_13.c", line 10854, in MetaInterp_vrefs_after_residual_call
  File "rpython_jit_metainterp_14.c", line 47038, in MetaInterp_stop_tracking_virtualref
Fatal RPython error: AssertionError
Abort trap: 6
(apenv) ~/git/ap-loadtester $
~~~

**Comment by pjenvey** [comment]
What version of pypy is this?

**Comment by rpappalax** [comment]
pypy-5.0.1-osx64

**Comment by pjenvey** [comment]
I'm keeping an eye out for this as I'm beginning to use the load tester more. My first suggestion is to upgrade to the latest PyPy 5.3.1 if possible as there's been a few crash fixes since 5.0.1.

**Comment by rpappalax** [comment]
Still seeing this error, but now I realize i forgot to upgrade my pypy:
https://pastebin.mozilla.org/8904198
